src/main/resources/default/taglib/t/pageHeader.html.pasta

```diff
@@ -24,13 +24,16 @@
                 <i:else>
                     <i:local name="markupTitle" value="@renderToString('title')"/>
                     <i:if test="isFilled(markupTitle)">
-                        <h1 class="legend mr-auto overflow-hidden text-nowrap text-truncate"><i:raw>@markupTitle</i:raw></h1>
+                        <h1 class="legend mr-auto overflow-hidden text-nowrap text-truncate">
+                            <i:raw>@markupTitle</i:raw>
+                        </h1>
                     </i:if>
                 </i:else>
             </i:if>
-            <i:render name="actions" />
-            <t:additionalActions labelKey="template.html.additionalActions" class="@('ml-1 ' + additionalActionsClass)" labelClass="d-none d-xl-inline-block">
-                <i:render name="additionalActions" />
+            <i:render name="actions"/>
+            <t:additionalActions labelKey="template.html.additionalActions" class="@('ml-1 ' + additionalActionsClass)"
+                                 labelClass="d-none d-xl-inline-block text-nowrap">
```

Comment:
I'm a huge fan of having one line per attribute, if they don't all fit into a single line
```suggestion
            <t:additionalActions class="@('ml-1 ' + additionalActionsClass)"
                                 labelKey="template.html.additionalActions"
                                 labelClass="d-none d-xl-inline-block text-nowrap">
```